en/components/rating.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Rating - Design System Component
+_description: The Rating component lets users apply a rating to an item or experience, such as an image, a forum post, an item for sale in a marketplace, and more.
+_keywords: Design Systems, Design Systems UX, UI kit, Figma, Figma to Angular, Export code from Figma, Figma to HTML, Figma UI kits, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Rating
+
+The Rating component enables users to apply a rating to an item or experience, such as a product, service, page, other entity, and more. The Rating can display a single user's, or an aggregate, star rating and it's used to let users set their own star rating.
+
+## Rating Demo
+
+<img class="responsive-img" src="../images/rating_demo.png" srcset="../images/rating_demo@2x.png 2x" />
+
+## Using the Rating in Figma
+
+The Rating is built from Label and Value, where the Value is the number of the rating scale. The number of icons in the rating scale is predefined up to five but if you need to design a Rating with more rating icons, the easiest way to do so is by inserting the component into your workspace. After that, right-clicking on the component will open a contextual menu where `Detach Instance` should be selected, and you should see the following detached layer structure:
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 componentVersion | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| 🚫 metadata         | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| Label              |                                                                                                                 |
```

Comment:
Missing info for the Label

Reply:
In my opinion, there is no need to add a description for the label, but I could add it for more clarity

Reply:
Added information for the label in the next commit